pages/1.10/installing/ent/custom/configuration/configuration-parameters/index.md

```diff
@@ -89,6 +90,17 @@ This topic provides configuration parameters available for [DC/OS Enterprise](ht
 | [zk_master_credentials](#zk-master)          | [enterprise type="inline" size="small" /] The ZooKeeper master credentials.  |
 | [zk_agent_credentials](#zk-agent)           | [enterprise type="inline" size="small" /] The ZooKeeper agent credentials.  |
 
+[enterprise]
+### adminrouter_auth_cache_enabled
+[/enterprise]
+
+_This option was added in DC/OS 1.10.6._
```

Comment:
1.10.6 is scheduled to ship on Apr 10. You might need to omit this info. (and add it later), if you plan to merge the PR now.